openhands/agent_server/sockets.py

```diff
@@ -39,13 +39,9 @@ async def events_socket(
     conversation_id: UUID,
     websocket: WebSocket,
     session_api_key: Annotated[str | None, Query(alias="session_api_key")] = None,
+    replay_all: Annotated[bool, Query()] = False,
```

Comment:
```suggestion
    restore_all: Annotated[bool, Query()] = False,
```

Nit: we have/had a _replay_ functionality, which actually executes the actions again in a new runtime environment. It's a bit confusing to name two different things with the same name. I know it comes from some code in V0, but maybe we could name it like, `restore` (as in, restore conversation history, as used elsewhere too), or maybe `resend` ?